code/modules/projectiles/guns/projectile/shotgun/ks23.dm

```diff
@@ -0,0 +1,21 @@
+/obj/item/gun/projectile/shotgun/pump/ks
+	name = "KS-23"
+	desc = "Karabin Spetsnaliy, desgined by Excelsior for a cheap crowd control for their agents, made with refused anti-aircraft barrels"
```

Comment:
```suggestion
	desc = "Karabin Spetsialniy, designed by Excelsior for cheap crowd control for their agents, made with reuused anti-aircraft barrels."
```
Do we even consider aircraft existing?

Reply:
not reused, and i wrote too fast i wanted rejected, i can change the entire description if you want

Reply:
> Do we even consider aircraft existing? I suppose there are some on terrestial planets, but Excelsior aren't really fond of them

Looking at expeditionary fleet equipment snd vehicle page they do exist